terraform/mesh_aws/README.md

```diff
@@ -0,0 +1,36 @@
+# Mesh Lambdas
+
+A terraform module to provide AWS infrastructure capable of sending and recieving Mesh messages
+
+## Configuration
+
+Example configuration required to use this module
+
+```
+module "mesh" {
+  source = "git::https://github.com/nhsdigital/spine-core-aws-common.git//terraform/mesh_aws?ref=v0.0.4"
+
+  name_prefix = "example-project-that-needs-mesh"
```

Comment:
think "-mesh" is being added to the name prefix elsewhere, so maybe not have an example which ends "-mesh"